docs/components.html

```diff
@@ -762,6 +762,41 @@ <h3 class="sub-section-title">Shadow</h3>
 
         </div>
 
+        <div id="navigation" class="section">
+
+                <h2 class="section-title">Navigation</h2>
+
+                <h3 class="sub-section-title">Classic Navbar</h3>
+
+                <div class="component-container navbar-container">
+
+                    <div class="navbar bg-red">
+    
+                        <div class="navbar-header">
+                
+                            <a href="#"><p class="navbar-header-title text-l text-white">Navbar Title</p></a>
+                            
+                        </div>
+                
+                        <div class="navbar-links">
+                
+                            <a href="#"><img src="../../../assets/download_white_36dp.svg" alt="download"></a>
+                            <a href="#"><img src="../../../assets/print_white_36dp.svg" alt="print"></a>
+                            <a href="#"><img src="../../../assets/bookmark_white_36dp.svg" alt="bookmark"></a>
+                
+                        </div>
+                        
+                    </div>
+
+                </div>
+
+                <div class="code-snippets">
+                    <iframe src="https://carbon.now.sh/embed?bg=rgba%28255%2C255%2C255%2C1%29&t=seti&wt=none&l=htmlmixed&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cnav%2520class%253D%2522navbar%2520bg-red%2522%253E%250A%2520%2520%2520%2520%250A%2520%2520%2520%2520%253Cdiv%2520class%253D%2522navbar-header%2522%253E%250A%250A%2520%2520%2520%2520%2520%2520%2520%2520%253Ca%2520href%253D%2522%2523%2522%253E%253Cp%2520class%253D%2522navbar-header-title%2520text-l%2520text-white%2522%253ENavbar%2520Title%253C%252Fp%253E%253C%252Fa%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%250A%2520%2520%2520%2520%253C%252Fdiv%253E%250A%250A%2520%2520%2520%2520%253Cdiv%2520class%253D%2522navbar-links%2522%253E%250A%250A%2520%2520%2520%2520%2520%2520%2520%2520%253Ca%2520href%253D%2522%2523%2522%253E%253Cimg%2520src%253D%2522...%2522%2520alt%253D%2522icon-desc%2522%253E%253C%252Fa%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%253Ca%2520href%253D%2522%2523%2522%253E%253Cimg%2520src%253D%2522...%2522%2520alt%253D%2522icon-desc%2522%253E%253C%252Fa%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%253Ca%2520href%253D%2522%2523%2522%253E%253Cimg%2520src%253D%2522...%2522%2520alt%253D%2522icon-desc%2522%253E%253C%252Fa%253E%250A%250A%2520%2520%2520%2520%253C%252Fdiv%253E%250A%2520%2520%2520%2520%250A%253C%252Fnav%253E" style="width: 884px; height: 485px; border:0; transform: scale(1); overflow:hidden;" sandbox="allow-scripts allow-same-origin"> </iframe>
+
+                </div>
+
+        </div>
```

Comment:
Remove the extra space and format the file

Reply:
done